### PR TITLE
Adding "will" as an author and adding What is Redis page

### PR DESCRIPTION
**Pull request by wjohnsto**

### CODE REVIEW

File: docs/explore/what-is-redis/index-what-is-redis.mdx [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+### Flexible Data Structures
+
+Redis is a multi-model database, and provides several built-in data structures such as:
+
+- Strings - any text or binary data (512MB max)
+- Hashes - field-value pairs that most commonly represent objects
+- Lists - a collection of Strings ordered by when they were added as a linked list. Useful for queues and "latest updates" for social media posts
+- Sets - an unordered collection of Strings with the ability to intersect, union, and diff against other Sets
+- Sorted Sets - a mix between a Set and a Hash, where the Set is ordered by a value (score). Useful for leaderboards, games, and analytics.
```
Comment (simonprickett) [inline docs/explore/what-is-redis/index-what-is-redis.mdx]: @wjohnsto @ajeetraina this is a strange way to describe a sorted set.  I'd drop the part about a Hash.  Probably best to say something like "Sorted sets - similar to a Redis Set, the Sorted Set is a collection of unique string members.  In a Sorted Set, each member is associated with a score that can be used to sort the collection".

File: docs/explore/what-is-redis/index-what-is-redis.mdx [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+- Bitmaps - not necessarily a data type, but a set of bit-oriented operations on the String type
+- Hyperloglogs - a probabilistic data structure used in order to count unique things (cardinality of a set)
+- Geospatial - a Sorted Set of longitude/latitude/name key-value pairs useful for maps, geosearching, and "nearby" features
+- Streams - a log data structure most useful as a message queue, often implemented as a file open in append-only mode
```
Comment (simonprickett) [inline docs/explore/what-is-redis/index-what-is-redis.mdx]: @wjohnsto @ajeetraina I'd drop the file reference here and say `Streams - a data type that models an append only log and which can be used as a durable message queue` (durable cos pub/sub isn't).

File: docs/explore/what-is-redis/index-what-is-redis.mdx [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+
+#### Redis Modules
+
+Redis also supports custom data structures called modules that run natively alongside core Redis. These modules can be used to create custom data structures that are not available in the built-in data structures. Examples of custom modules include:
+
+- [RediSearch](https://redis.com/redis-enterprise/redis-search/) - a real-time search engine that runs on your Redis dataset and allows you to query data that has just been indexed
```
Comment (simonprickett) [inline docs/explore/what-is-redis/index-what-is-redis.mdx]:
```suggestion
- [RediSearch](https://redis.com/redis-enterprise/redis-search/) - a real-time search and secondary indexing engine that runs on your Redis dataset and allows you to query data that has just been indexed
```

File: docs/explore/what-is-redis/index-what-is-redis.mdx [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+- [RedisTimeSeries](https://redis.com/redis-enterprise/redis-time-series/) - a time series data type with capabilities like automatic downsampling, aggregations, labeling and search, compression, and enhanced multi-range queries as well as built-in connectors to popular monitoring tools like Prometheus and Grafana enable the extraction of data into useful formats for visualization and monitoring
+- [RedisBloom](https://redis.com/redis-enterprise/redis-bloom/) - provides Redis with support for additional probabilistic data structures and allow for constant memory space and extremely fast processing while still maintaining a low error rate. Supports Bloom and Cuckoo filters to determine whether an item is present or absent from a collection with a given degree of certainty, Count-min sketch to count the frequency of the different items in sub-linear space, and Top-K to count top k events in a near deterministic manner
```
Comment (simonprickett) [inline docs/explore/what-is-redis/index-what-is-redis.mdx]:
```suggestion
- [RedisBloom](https://redis.com/redis-enterprise/redis-bloom/) - provides Redis with support for additional probabilistic data structures and allows for constant memory space and extremely fast processing while still maintaining a low error rate. Supports Bloom and Cuckoo filters to determine whether an item is present or absent from a collection with a given degree of certainty, Count-min sketch to count the frequency of the different items in sub-linear space, and Top-K to count top k events in a near deterministic manner
```

File: docs/explore/what-is-redis/index-what-is-redis.mdx [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+
+While Redis open-source was not built to be a durable and consistent database, [Redis Enterprise provies a durability and consistency](https://redis.com/redis-enterprise/technology/durable-redis/) for Redis and allows you to use Redis as both a cache and a database.
+
+### High availability and scalability
+
+Redis provides a primary-replica architecture as a single node or cluster. This allows you to build highly available, scalable, and reliable applications on top of Redis. You can easily scale Redis up, down, in, and out to meet application demands.
+
+### Open Source
+
+Redis is open-source and available for free, and offers open-source clients in many languages. The [Redis Modules listed above](#redis-modules) are also available for download or use in Redis Enterprise Cloud.
```
Comment (simonprickett) [inline docs/explore/what-is-redis/index-what-is-redis.mdx]:
```suggestion
Redis is open source and available for free, and offers open-source clients in many languages. The [Redis Modules listed above](#redis-modules) are also available for download or use in Redis Enterprise Cloud.
```

File: docs/explore/what-is-redis/index-what-is-redis.mdx [hunk 6]
```diff
@@ -0,0 +1,155 @@
...
+
+### Fraud Detection
+
+Redis is built to handle real-time AI and machine learning workloads because of its scalability and high write throughput at low latency. Redis is often used as a primary database, enabling deep learning models directly where the data lives. Bloom filters, time series, and other data structures that work natively with Redis enable cost reduction with high-speed statistical analysis. Learn more about [Redis fraud detection](https://redis.com/solutions/use-cases/fraud-detection/).
+
+### Real-time Inventory
+
+Retailers need to ensure that their real-time inventory systems can survive seasonal peaks, maintain data consistency, and deliver instant results. Redis is a great choice for this use case. It is a highly-available and highly-scalable database that can handle millions of queries per second. Redis clusters can be configured to replicate across multiple servers in a highly-available manner, enabling data consistency between stores. Learn more about [Redis inventory](https://redis.com/solutions/use-cases/real-time-inventory/).
```
Comment (simonprickett) [inline docs/explore/what-is-redis/index-what-is-redis.mdx]:
```suggestion
Retailers need to ensure that their real-time inventory systems can survive seasonal peaks, maintain data consistency, and deliver instant results. Redis is a great choice for this use case. It is a highly-available and highly-scalable database that can handle millions of queries per second. Redis clusters can be configured to replicate across multiple servers in a highly-available manner, enabling data consistency between stores. Learn more about [Redis for real-time inventory management](https://redis.com/solutions/use-cases/real-time-inventory/).
```

File: docs/explore/what-is-redis/index-what-is-redis.mdx [hunk 7]
```diff
@@ -0,0 +1,155 @@
...
+
+### Fast data ingest
+
+Redis can handle millions of read/write operations per second at sub-millisecond latencies, and it runs on AWS, GCP, Azure, and other cloud platforms. This makes Redis a great choice for processing large volumes of data that arrive in bursts, data from multiple sources/formats, data that needs to be filtered and analyzed, and data that is distributed geographically. Learn more about [Redis data ingestion](https://redis.com/solutions/use-cases/fast-data-ingest/).
+
+## Redis Language Support
+
+Redis supports most high-level popular programming languages and has SDKs built to make it easy to get started. Redis supports the following languages:
```
Comment (simonprickett) [inline docs/explore/what-is-redis/index-what-is-redis.mdx]:
```suggestion
Redis supports most high-level popular programming languages and has SDKs built to make it easy to get started. Redis clients are available for the following languages and more:
```

File: docs/explore/what-is-redis/index-what-is-redis.mdx [hunk 8]
```diff
@@ -0,0 +1,155 @@
...
+- JavaScript
+- Node.js
+- Java
+- Go
+- C/C++
+- C#
+- PHP
+- Ruby
+- Perl
+
```
Comment (simonprickett) [inline docs/explore/what-is-redis/index-what-is-redis.mdx:137]:
```suggestion
- Rust
```

File: docs/explore/what-is-redis/index-what-is-redis.mdx [hunk 9]
```diff
@@ -0,0 +1,155 @@
...
+## Redis vs. Memcached
+
+Both Redis and Memcached are open-source, powerful, in-memory data stores. The main difference between the two is that Redis is a more full-featured database that is built to fit a number of different use-cases. Memcached is primarily used for key/value caching. Redis is used for both caching and as a database.
+
+## Using Redis on AWS, GCP, or Azure
+
+There are a number of ways to use Redis on AWS, but the recommended way is using Redis Enterprise Cloud.
```
Comment (simonprickett) [inline docs/explore/what-is-redis/index-what-is-redis.mdx]: @wjohnsto @ajeetraina This seems like a strange section that doesn't add value or talk about GCP or Azure.  Remove it?

**Review by simonprickett — APPROVED** [review]
@wjohnsto left some comments to think about as take them or leave them items, approved.

**Review by simonprickett — APPROVED** [review]
@ajeetraina @wjohnsto re-reviewed looks good to me - merge when ready?

**Review by ajeetraina — APPROVED** [review]
Looks good to me.

**Comment by simonprickett** [comment]
Fixed conflict, merged.

**Pull request merged**